cmd/fs-v1-metadata.go

```diff
@@ -269,6 +309,17 @@ func loadFormatFS(fsPath string) (*formatConfigV1, error) {
 		return nil, err
 	}
 
+	// Check if format is currently supported.
+	if !isFSValidFormat(format) {
+		return nil, errCorruptedFormat
+	}
+
+	// Check for format version is current.
+	if format.FS.Version != fsFormatVersion {
+		return nil, errFSFormatOld
+	}
```

Comment:
Better do these checks version in the caller `loadFormatFS()` because as a code reader I expected loadFormatFS to just load it and not do validation of versions.